docs/BEMAN_STANDARD.md

```diff
@@ -133,6 +133,12 @@ contain a one- or two-paragraph summary describing the library's purpose.
 [Give *std::optional* Range Support (P3168R1)](https://wg21.link/P3168R1).
 ```
 
+**[README.LIBRARY_STATUS]**  RECOMMENDATION: Following the implements section and a newline, the `README.md` should state its status of development. Possible states: `BEMAN DEVELOPMENT`, `BEMAN UNSTABLE`, `BEMAN STABLE`. Defaults to `BEMAN DEVELOPMENT` state (if the section is missing). 
```

Comment:
```suggestion
```suggestion
**[README.LIBRARY_STATUS]**  RECOMMENDATION: Following the implements section and a newline, the `README.md` should state its status of development. Possible library states: 
    * `BEMAN DEVELOPMENT`: The target paper(s) may change within the ISO standardization stages, thus also the implementation.
     * `BEMAN UNSTABLE`: The library implementation matches the paper(s), but needs more testing and possible more updates to be Beman Standard conformant. 
     * `BEMAN STABLE`: All target papers were accepted into the ISO C++ Working Draft and all of them were implemented in current Beman library. The code is production ready. 
     
Defaults to `BEMAN DEVELOPMENT` state (if the section is missing). 


```